zowe-cli-id-federation-plugin/src/api/templates/tss.jcl

```diff
@@ -0,0 +1 @@
+TSS ADD({{mainframe_id}}) IDMAP(zwednmap) IDMAPDN('{{#escape}}{{distributed_id}}{{/escape}}') IDMAPRN('{{#escape}}{{registry}}{{/escape}}') IDLABEL('dn user mapping')
```

Comment:
For the IDLABEL, I would suggest to use the `user_name` value from the CSV. To be consistent with RACF

Reply:
done